network/android/src/main/java/com/capacitorjs/plugins/network/NetworkStatus.java

```diff
@@ -0,0 +1,14 @@
+package com.capacitorjs.plugins.network;
+
+public class NetworkStatus {
+
+    public enum ConnectionType {
```

Comment:
you can use an enum that also contains the string values, like https://github.com/ionic-team/capacitor-plugins/blob/d97898662d0ba037e5f8448990a91de5ec6a4234/camera/android/src/main/java/com/capacitorjs/plugins/camera/CameraSource.java

Then in parseNetworkStatus you can avoid using a switch for converting the value to strings